extension/src/panel.ts

```diff
@@ -62,21 +61,22 @@ export class HistoryDebuggerPanel {
   public static readonly viewType = "temporal-debugger-plugin"
 
   private readonly panel: vscode.WebviewPanel
-  // TODO: this isn't populated, make sure everything is properly disposed
   private disposables: vscode.Disposable[] = []
   private updateWorkflowTaskHasBreakpoint = (_hasBreakpoint: boolean) => {
     // noop, to be set in the updateCurrentWFTStarted handler
   }
 
+  private timeouts: NodeJS.Timeout[] = []
+
   show(): void {
     this.panel.reveal(vscode.ViewColumn.Beside)
   }
 
   async updateCurrentWFTStarted(eventId: number): Promise<void> {
     const p = new Promise<boolean>((resolve, reject) => {
       this.updateWorkflowTaskHasBreakpoint = resolve
-      // TODO: clear timeout if disposed
-      setTimeout(() => reject(new Error("Timed out waiting for response from webview")), 5000)
+      const timeoutObj = setTimeout(() => reject(new Error("Timed out waiting for response from webview")), 5000)
+      this.timeouts.push(timeoutObj)
```

Comment:
Not sure this is how I'd do it.
There's no reason to keep the timeout around after it is resolved.
Probably no need to clear after disposing, not sure why that comment was here.

Reply:
Removed this as I was confused by that comment too 😅 Maybe the comment had more to do with immediately erroring if the panel is closed and this promise is not resolved?